.changeset/twelve-actors-train.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/ui': minor
```

Comment:
@Majid-J Could you please change this to a patch update?

Reply:
Smoke test locally and found a nit. It seems `'Enter your email'` should be `Enter your Email` with email capitalized but otherwise it looks good to me

![Screenshot 2023-05-17 at 1 09 35 PM](https://github.com/aws-amplify/amplify-ui/assets/40295569/e5127272-102d-419e-b605-657db9bd5cc0)